DataAccessLibrary/TextFileDataAccess.cs

```diff
@@ -0,0 +1,57 @@
+﻿using DataAccessLibrary.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+using System.Linq;
+
+namespace DataAccessLibrary
+{
+    public class TextFileDataAccess
+    {
+        public List<SingleRowModel> ReadAllRecords(string textFile)
+        {
+            if (File.Exists(textFile) == false)
+            {
+                return new List<SingleRowModel>();
+            }
+
+            var lines = File.ReadAllLines(textFile);
+            List<SingleRowModel> output = new List<SingleRowModel>();
+
+            foreach (var line in lines)
+            {
+                SingleRowModel c = new SingleRowModel();
```

Comment:
"Clean Coding" says to call the object what it is i.e. singleRowModel (or maybe 'row'?)